nexus/src/external_api/http_entrypoints.rs

```diff
@@ -8952,6 +9059,34 @@ async fn session_sshkey_create(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// Create an SSH public key
```

Comment:
is it weird that we have this in the API?

Reply:
Is it weird? I don't think so. Maybe labeled improperly? It's not actually _creating_ the key so much as creating a record of the key which can be used later to access an instance. Should we be doing something different?